.github/new_pull_request.md

```diff
@@ -0,0 +1,16 @@
+### Problem Statement
+
+
+### Solution
+
+
+### PRT test Cases
+
+
```

Comment:
I don't think we need this section. If anything we should add a comment only visible when opening the PR that gives an example PRT comment and explains how to use it.